src/partition/repart.c

```diff
@@ -1965,6 +1971,8 @@ static int partition_read_definition(Partition *p, const char *path, const char
                 { "Partition", "VerityHashBlockSizeBytes", config_parse_block_size,        0, &p->verity_hash_block_size  },
                 { "Partition", "MountPoint",               config_parse_mountpoint,        0, p                           },
                 { "Partition", "EncryptedVolume",          config_parse_encrypted_volume,  0, p                           },
+                { "Partition", "Compression",              config_parse_string,            0, &p->compression             },
+                { "Partition", "CompressionLevel",         config_parse_string,            0, &p->compression_level       },
```

Comment:
Setting CONFIG_PARSE_STRING_SAFE_AND_ASCII is better for safety? As these arguments are directly passed to mkfs. We should not provide fuzzing surface for mkfs.